news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md

```diff
@@ -0,0 +1,183 @@
+
+
+### Tests
+
+* [#14491](https://github.com/PrestaShop/PrestaShop/pull/14491): Disable Selenium Guest scenario as it fails randomly, by [@matks](https://github.com/matks)
+
+
+## Code changes in the "1.7.6.x" branch (for v1.7.6.0)
+
+### Core
+
+* [#14438](https://github.com/PrestaShop/PrestaShop/pull/14438): Testing the factory with number and price specifications, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#14453](https://github.com/PrestaShop/PrestaShop/pull/14453): Update facetedsearch to 3.0.6, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#14503](https://github.com/PrestaShop/PrestaShop/pull/14503): Update changelog for 1.7.6.0 RC 2, by [@matks](https://github.com/matks)
+
+
+### Back office
+
+* [#14460](https://github.com/PrestaShop/PrestaShop/pull/14460): Ignore index.php files in Doctrine entities scanning, by [@jolelievre](https://github.com/jolelievre)
+
+
+### Front office
+
+* [#14444](https://github.com/PrestaShop/PrestaShop/pull/14444): Check if selected address belongs to user, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### Tests
+
+* [#14467](https://github.com/PrestaShop/PrestaShop/pull/14467): Remove unwanted e2e tests, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+
+## Code changes in modules, themes & tools
+
+### Google sitemap
+
+* [#106](https://github.com/PrestaShop/gsitemap/pull/106): Do not call getIsset() with getValue(), by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#107](https://github.com/PrestaShop/gsitemap/pull/107): Add all images of a product to the sitemap, instead of only the cover image. Thank you [@Pixep](https://github.com/Pixep)
+
+
+### Nightly-board
+
+* [#9](https://github.com/PrestaShop/nightly-board/pull/9): Disable reports, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### Gamification
+
+* [#61](https://github.com/PrestaShop/gamification/pull/61): Fix the deprecated create_function for PHP 7.2, by [@eternoendless](https://github.com/eternoendless)
+* [#67](https://github.com/PrestaShop/gamification/pull/67): Undefined variable when running in fancybox, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#68](https://github.com/PrestaShop/gamification/pull/68): Bump version to 2.3.1, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#69](https://github.com/PrestaShop/gamification/pull/69): Release 2.3.1, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### Example Module Mailtheme
+
+* [#1](https://github.com/PrestaShop/example_module_mailtheme/pull/1): Update modules after changes from the core, use two different templat…, by [@jolelievre](https://github.com/jolelievre)
+* [#2](https://github.com/PrestaShop/example_module_mailtheme/pull/2): Update min version, by [@jolelievre](https://github.com/jolelievre)
+
+
+### CsaGuzzleBundle
```

Comment:
Maybe we can explain that this is a fork we created to have a php7.3 compliant version of this lib ?

Reply:
should we do that in an article ? or, maybe, is it clear enough on the readme.md of our fork?